che-theia-factory/src/browser/factory-theia-client.ts

```diff
@@ -161,11 +167,12 @@ export class FactoryTheiaClient implements FrontendApplicationContribution {
             }));
 
         });
-        Promise.all(importProjectPromises).then(() => {
```

Comment:
Refactoring +
`await new Promise(resolve => setTimeout(resolve));`.
Because time to time I had an incident with a non-existent file exception during open file action immediately after import.